### PR TITLE
Add Vortex extension

### PR DESCRIPTION
**Pull request by stackingsaunter**
## Description

Extension allows to control you bitcoin lightning wallet directly from Raycast through [NWC connection 
](https://nwc.dev/)

### Features
- Pay lightning invoices
- Send to lightning address
- Create invoices
- View wallet balance
- View transaction list

## Screencast

vortex 2024-02-18 at 00 43 12

vortex 2024-02-18 at 00 45 32

vortex 2024-02-18 at 00 45 59

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/vortex/package.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        }
+      ],
+      "required": true,
+      "default": "USD"
+    }
+  ],
+  "commands": [
+    {
+      "name": "send",
+      "title": "Send",
```
Comment (pernielsentikaer) [inline extensions/vortex/package.json:45]:
```suggestion
      "title": "Send to Address or Invoice",
```

File: extensions/vortex/package.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      "title": "View Balance",
+      "keywords": ["balance", "wallet", "lightning", "vortex", "bitcoin"],
+      "description": "Display your current wallet balance",
+      "icon": "vortex.png",
+      "mode": "view",
+      "script": "src/ShowBalance.tsx"
+    },
+    {
+      "name": "createinvoice",
+      "title": "Receive",
```
Comment (pernielsentikaer) [inline extensions/vortex/package.json:72]:
```suggestion
      "title": "Receive",
      "subtitle": "Vortex",
```

**Comment by stackingsaunter** [comment]
Hey @pernielsentikaer, by screencast you mean a recording of how it works?

**Comment by stackingsaunter** [comment]
Hey @pernielsentikaer I added what you asked and here's a video how I use it

https://www.youtube.com/watch?v=0t42wYwuuQI

Here's receive flow I forgot to record:


https://github.com/raycast/extensions/assets/68239231/a3baafcf-2bf6-4af0-8726-0e3cce62c50b


I can also give you an invite code for Alby Account and send you some sats so you can play with it yourself If you want :) Let me know!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**